docs/user_guide/assessment/common_fairness_metrics.rst

```diff
@@ -299,7 +305,128 @@ capture the costs of missclassification disparities.
 
 
 
+.. _assessment_four_fifths:
+
+The Four Fifths Rule: Often Misapplied
+--------------------------------------
+
+In the literature around fairness in machine learning, one will often find
+the so-called "four fifths rule" or "80% rule" used to assess whether a model
+(or mitigation technique) has produced a 'fair' result.
+Typically, the rule is implemented by using the demographic parity ratio introduced
+in the :ref:`assessment_demographic_parity` section above, with a result considered
+'fair' if the ratio exceeds 80% for all identified subgroups.
+*Application of this threshold is wrong in many scenarios.*
+
+As we note in many other places in the Fairlearn documentation, 'fairness'
+must be assessed by examining the entire sociotechnical context of a machine
+learning system.
+In particular, it is important to start from the harms which can occur to real
+people, and work inwards towards the model.
+The demographic parity ratio is simply a metric by which a particular model
+may be measured (on a particular dataset).
+Given the origin of the 'four-fifths rule' (which we will discuss next), its
+application may also give an unjustified feeling of legal invulnerability by
+conflating fairness with legality.
+In reality, 'fairness' is not always identical to 'legally allowable,' and
+the former may not even be a strict subset of the latter. [#f1]_
+
+The 'four fifths rule' has its origins in a specific area of US
+federal employment law.
+It is a limit for
+`prima facie evidence <https://en.wikipedia.org/wiki/Prima_facie>`_
+that illegal discrimination has occurred relative to a 
+relevant control population.
```

Comment:
```suggestion
relevant control population. The four-fifths rule is one of many test statistics that can be used to establish a prima facie case, but it is generally only used within the context of employment regulation. 
```

My intent with this suggestion is to further narrow the scope of when the four fifths rule is generally used/reasonable to use. We could also cite the [Uniform Guidelines on Employee Selection Procedures (1978)](https://www.ecfr.gov/current/title-29/subtitle-B/chapter-XIV/part-1607) so readers can get more info.

Reply:
Accepted, with the addition of 'US Federal'